x-pack/package.json

```diff
@@ -147,6 +148,7 @@
     "chroma-js": "^1.3.6",
     "classnames": "2.2.5",
     "concat-stream": "1.5.1",
+    "cookie": "^0.3.1",
```

Comment:
I'm surprised we haven't needed to do any cookie parsing up until this point. I assume that HAPI doesn't provide any facilities for this that we can use?

Reply:
I wasn't able to find any unfortunately...